app/components/timeline_entry/component.rb

```diff
@@ -199,5 +199,15 @@ def stage_changed_vars
           ).strip,
       }
     end
+
+    def application_declined_vars
+      if timeline_event.application_form.declined_at?
+        render(
+          StatusTag::Component.new("declined", class_context: "timeline-event"),
+        ).strip
+      else
+        ""
+      end
+    end
```

Comment:
```suggestion
    def application_declined_vars
      {}
    end
```

I don't think we're using any variables from this method in the view so we can return an empty hash here.